src/components/Api.js

```diff
@@ -58,9 +58,7 @@ export default class Api {
   deleteCard(cardID) {
     return fetch(this._baseUrl + "/cards/" + cardID, {
       method: "DELETE",
-      headers: {
-        authorization: "a9ab881f-c455-4a3d-9bb8-b4ec04211711",
```

Comment:
Please, use `headers: this._headers,`  everywhere https://disk.yandex.ru/i/uOhjHfojc6zk8Q

Reply:
Addressed now.